website/.unbroken_exclusions

```diff
@@ -1,6 +1,7 @@
 !node_modules
 !blog/2019-*
 !blog/2020-*
+!versioned_docs/version-0.63/*-api-windows*.md

```

Comment:
Can you update the fix-unbroken.js script with these exclusions?

https://github.com/microsoft/react-native-windows-samples/blob/66e09b9da02690c360ec40716091463a6ced7c31/website/fix-unbroken.js#L70

Reply:
Otherwise the next time we run the script, it will get lost.

Reply:
yeah; maybe we just always insert `!versioned_docs/*/*-api-windows*.md` ?